### ISSUE TITLE
Update the repo team ownership according to the CODEOWNERS/MAINTAINERS files

### ISSUE THREAD

**Issue by toddysm** [notaryproject/.github#4]
Currently there is no github team with permissions to maintain this repository. For the purpose of segregation of duties, I propose to have repo ownership team with members listed in the CODEOWNERS/MAINTAINERS files. The proposal is to create a GitHub team with the name `notaryproject-governance-maintainers` with members from the above files and assigning this team ownership of the `.github` repository.

**Comment by yizha1** [comment]
LGTM

**Comment by SteveLasker** [comment]
I believe this is about:
- org maintainers
- sub-project maintainers

As the project evolved, additional repos have been added:
 - [notation-core-go](https://github.com/notaryproject/notation-core-go)
 - [notation-go](https://github.com/notaryproject/notation-go)
 - [notaryproject.dev](https://github.com/notaryproject/notaryproject.dev)
 - [meeting-notes](https://github.com/notaryproject/meeting-notes)
 - [.github](https://github.com/notaryproject/.github)
 - [roadmap](https://github.com/notaryproject/roadmap)

I'd suggest creating another issue to capture and clarify the relationship between:
- Notary v1 [notary](https://github.com/notaryproject/notary)
- Notary v2 (aka notation) (collection of repos)
- [TUF](https://github.com/notaryproject/tuf) incubation

And the additional repos that span the org:
- Org Governance - currently split between [notary](https://github.com/notaryproject/notary/blob/master/MAINTAINERS) and [.github]((https://github.com/notaryproject/.github)) PRs
- [Website content](https://github.com/notaryproject/notaryproject.dev), which spans the breadth of the project
- [Roadmap](https://github.com/notaryproject/roadmap), also meant to cover the breadth of the org project.

**Comment by priteshbandi** [comment]
LGTM

**Comment by vaninrao10** [comment]
LGTM

**Comment by iamsamirzon** [comment]
LGTM IANAM

**Comment by toddysm** [comment]
Based on the above approvals from active maintainers this is completed to unblock PR approvals for this repository.
